plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveWriteUtils.java

```diff
@@ -291,6 +294,11 @@ public static ObjectInspector getJavaObjectInspector(Type type)
     }
 
     public static Object getField(Type type, Block block, int position)
+    {
```

Comment:
This method is called in just one place (`io.trino.plugin.hive.util.HiveWriteUtils#createPartitionValues`), so update that place instead of adding an overload.

(in this case, the overload is not ideal, because you need to think which one to call, or you can accidentally call the wrong one)

Reply:
I added the overload because the method is public, and if anyone tries to use it in a non-parquet-related location, it seems like they shouldn't need to pass in the default time zone.

Reply:
public or not -- it doesn't matter. This is not a library where we need to care about backward compat of public API

Reply:
Right; fixed.